src/screens/RegisterCompleted.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import {
+  SafeAreaView,
+  View,
+  Text,
+  TouchableOpacity,
+  StyleSheet,
+  Image,
+} from 'react-native';
+import {Colors} from '../utils/Colors';
+import {REGISTER_COMPLETED_TEXT} from '../utils/Constants';
+
+const RegisterCompleted = () => {
+  return (
+    <SafeAreaView style={styles.mainContainer}>
+      <View style={styles.imageContainer}>
+        <Image
+          style={styles.heartHandsImage}
+          source={require('../assets/images/heart-handshake.png')}
+        />
+      </View>
+      <View style={styles.succesContainer}>
+        <View style={styles.mainText}>
+          <Text style={styles.title}>
+            {REGISTER_COMPLETED_TEXT.account_created}
+          </Text>
+          <View style={styles.textContainer}>
+            <Text style={styles.paragraphs}>
+              {REGISTER_COMPLETED_TEXT.thanks}
+            </Text>
+            <Text style={styles.paragraphs}>
+              {REGISTER_COMPLETED_TEXT.together}
+            </Text>
+          </View>
+        </View>
+        <TouchableOpacity
+          style={styles.btnAccept}
+          onPress={() => {
+            console.log('Register Completed');
+          }}>
+          <Text style={styles.textBtnAccept}>Iniciar</Text>
```

Comment:
I see you already have a constant file for text. Could you add this text to that file?